src/main/java/ch/njol/skript/conditions/CondCompare.java

```diff
@@ -381,15 +390,36 @@ public boolean check(final Event e) {
 			}
 		), isNegated());
 	}
-	
+
+	/**
+	 * Used to directly compare two lists for equality.
+	 * This method assumes that {@link CondCompare#first} and {@link CondCompare#second} are both non-single.
+	 * @param event the event with which to evaluate {@link CondCompare#first} and {@link CondCompare#second}.
+	 * @return Whether every element in {@link CondCompare#first} is equal to its counterpart in {@link CondCompare#second}.
+	 * 		e.g. (1,2,3) = (1,2,3), but (1,2,3) != (3,2,1)
+	 */
+	private boolean compareLists(Event event) {
+		Object[] first = this.first.getArray(event);
+		Object[] second = this.second.getArray(event);
+		boolean notMatch = isNegated();
+		// read 'notMatch' as false and '!notMatch' as true
```

Comment:
Personally not a fan of `notMatch`, any ideas on a better name? `not` just doesn't feel like it expresses `negated` to me. I thought of `shouldMatch` or just retaining a boolean value of `isNegated`.